locale/zh_CN.po

```diff
@@ -1932,11 +1908,11 @@ msgstr ""
 "水和二氧化碳转换成葡萄糖。葡萄糖的产出速率取决于二氧化碳的浓度。"
 
 #: ../src/gui_common/tooltip/ToolTipManager.tscn:1474
-#, fuzzy
 msgid "NITROGEN_FIXING_PLASTID_PROCESSES_DESCRIPTION"
 msgstr ""
-"固氮质体是一种可以用氮和氧还有ATP制造氨的蛋白质，氨对细胞生长至关重要。这个"
-"过程被称为有氧固氮。"
+"将[thrive:compound]ATP[/thrive:compound]转换成[thrive:compound]氨[/thrive:compound"
```

Comment:
@fgdfgfthgr-fox just FYI, this is incorrect, the compound names inside the bbcode tags are the internal names used in the code, they should not be translated, otherwise they are broken. So this should be `[thrive:compound]glucose[/thrive:compound]` for it to work.
I'm going to merge this PR anyway, but there's still time before the next release to fix the translations.